### PR TITLE
Fixes #20882: Adapt new rudder report to work with techniques

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/20882

### CODE REVIEW

File: tree/30_generic_methods/service_stopped.cf [hunk 1]
```diff
@@ -32,31 +32,37 @@ bundle agent service_stopped(name)
...
       "needs_stop" expression => "${report_data.method_id}_ncf_services_check_kept";
-
+    any::
       "pass3" expression => "pass2";
       "pass2" expression => "pass1";
       "pass1" expression => "any";
 
   methods:
-    pass2.!pass3::
+pass1.!pass2::
```
Comment (amousset) [inline tree/30_generic_methods/service_stopped.cf:43]: indentation

File: tree/20_cfe_basics/services.cf [hunk 2]
```diff
@@ -231,6 +231,8 @@ bundle agent ncf_services(service, action)
       "pass1" expression => "any";
 
   methods:
+  pass1.!pass2::
```
Comment (amousset) [inline tree/20_cfe_basics/services.cf:234]: indentation

**Comment by ncharles** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**